packages/@aws-cdk/aws-events-targets/lib/aws-api.ts

```diff
@@ -127,3 +130,37 @@ function awsSdkToIamAction(service: string, action: string): string {
   const iamAction = action.charAt(0).toUpperCase() + action.slice(1);
   return `${iamService}:${iamAction}`;
 }
+
+const sdkDocsUrl = 'https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/index.html';
+
+/**
+ * Check that the given service and actions exist in the AWS SDK. If the service name is given in an incorrect case
+ * e.g. rds instead of RDS, the normalized service name is returned.
+ * @param service Service name
+ * @param action Action name
+ * @param apiVersion API version
+ * @returns Normalized service name
+ * @throws {Error} The given service and/or action don't exist
+ */
+function checkServiceAndActionExists(service: string, action: string, apiVersion?: string): string {
+  let sdkService: any;
+  let normalizedServiceName: string;
+
+  if ((SDK as any)[service]) {
```

Comment:
I'd rather do a complete case-insensitive comparison.

Get a list of all services, map the lowercase name of the service to the actual case, and look up in there.

Reply:
Got it.